hunt/app/views/animals/new.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Animals#new</h1>
-<p>Find me in app/views/animals/new.html.erb</p>
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <div class="card">
+        <div class="card-header">Create an Animal</div>
+          <div class="card-body">
+
+            <%= simple_form_for @animal do |f| %>
+              <%= f.error_notification %>
+              <%= f.input :name, required: true, error: "Name can't be blank" %>
+              <%= f.input :state, collection:Animal::STATES, required: true, error: "State can't be blank"%>
+              <%= f.input :category, collection: Animal::CATEGORIES, required: true, error: "Category can't be blank"%>
+
+              Create an Animal<%=f.submit class: 'btn btn-dark'%>
```

Comment:
You actually don't need this extra text, `f.submit` will add it for you ✨ 

![Screen Shot 2020-08-04 at 3 39 11 PM](https://user-images.githubusercontent.com/7715500/89342623-aae07100-d668-11ea-96fc-7c915b54d733.png)